lib/import-export-visitor.js

```diff
@@ -593,17 +636,21 @@ function addToSpecifierMap(map, __ported, local) {
 
 function toModuleImport(source, specifierMap, uniqueKey) {
   const parts = ["module.importSync(", source];
-  const importedNames = specifierMap && Object.keys(specifierMap);
+  const importedNames = specifierMap ? Object.keys(specifierMap) : [];
```

Comment:
It's type consistency. I may be able to get away with explicit a `null` engines like consistent types.
I ran into perf issues in Firefox due to it and just saw [this](https://twitter.com/bmeurer/status/846951275480711168) yesterday. How often is specifier map provided (what's the common case).